[DE] Update `Help centre/Store` (#12341)
ppy/osu-wiki: wiki/Help_centre/Store/de.md

tags:
  - store
  - product
  - Produkt
  - keycap
  - keycaps
  - Taste
  - Tasten
  - deskmat
  - deskmats
  - Tischmatte
  - Unterlage
---

# osu!store und Merchandise

*Hauptseite: [Hilfecenter](/wiki/Help_centre)*\
*Für Hilfe zu älteren Produkten, siehe: [Shoparchiv](/wiki/Help_centre/Store/Store_archive)*

Dieser Artikel bezieht sich nur auf Produkte, die im [osu!store](https://osu.ppy.sh/store/listing) erworben werden können.

## Ich möchte meine Anschrift ändern! {id=address}

Sende eine E-Mail mit der Bestellnummer und der neuen Adresse an [merch@ppy.sh](mailto:merch@ppy.sh).

> Wenn deine Bestellung noch nicht versandt wurde, aktualisieren wir die Adresse und senden die Bestellung an die neue Anschrift. Wenn die Bestellung bereits unterwegs ist, können keine Änderungen mehr vorgenommen werden.

Reviewer: Bit less formal maybe

```suggestion
Wenn deine Bestellung noch nicht versandt wurde, aktualisieren wir die Adresse und senden die Bestellung an die neue Anschrift. Wenn die Bestellung bereits unterwegs ist, können wir leider nicht mehr viel machen.
```

.. is `machen` correct there? maybe `tun`

Author: According to [this article](https://dein-sprachcoach.de/unterschied-zwischen-machen-und-tun/), `tun` is used for formal texts and `machen` in informal conversations. `Tun` is also used for emphasising specific actions.

You can say both, though my gut feeling tells me that `etwas für jemanden tun` is more common then `etwas für jemanden machen`.

Reviewer: I was proposing the less formal variant on purpose after looking at the English text. Both are fine, but the current version seems a bit stiff.

## Mein Produkt wurde noch nicht geliefert! {id=status}

Beim Kauf eines Produkts aus dem osu!store erhältst du eine E-Mail mit einer Sendungsverfolgung, sobald das Produkt versandt wurde.

Wenn deine Bestellung in deinem Land ankam, aber noch nicht geliefert wurde, wird sie möglicherweise bei deinem lokalen Postamt aufbewahrt. Wende dich an die Poststelle und nenne deine Sendungsnummer für weitere Details! Das ist für internationale Käufe, bei denen Zollgebühren oder andere Abgaben anfallen, manchmal erforderlich.

> Gelegentlich bleiben Pakete im Versandsystem stecken. Wenn deine Bestellung irgendwo in den Vereinigten Staaten festsitzt, kann sie möglicherweise mit [USPS](https://faq.usps.com/s/article/USPS-Tracking-The-Basics) nachverfolgt werden. Dazu kann normalerweise die zuständige Poststelle kontaktiert werden. Wenn die Bestellung außerhalb der Vereinigten Staaten festsitzt, kontaktiere den Postdienst deines Landes für weitere Hilfe.

Author: I wondered as well while translating.

A German postal service will only make tracking available after receiving the packet, don't they?

Including a link to DHL or something (and mention USPS instead on the side) sounds like a good idea to me.

> Gelegentlich bleiben Pakete im Versandsystem stecken. Wenn deine Bestellung irgendwo in den Vereinigten Staaten festsitzt, kann sie möglicherweise mit [USPS](https://faq.usps.com/s/article/USPS-Tracking-The-Basics) nachverfolgt werden. Dazu kann normalerweise die zuständige Poststelle kontaktiert werden. Wenn die Bestellung außerhalb der Vereinigten Staaten festsitzt, kontaktiere den Postdienst deines Landes für weitere Hilfe.

Reviewer: I don't know enough about the osu store to say either way. What I *can* tell you is that DHL does have tracking for anything "Päckchen" and above (Paket, Speditionslieferung) but not for normal Briefe or Postkarten. So unless the store products are sent in a normal letter envelope, there should be some kind of tracking. That said, I have no clue which logistics company/Spedition is used. There's also Hermes, DPD, GLS, UPS and some others.

> Gelegentlich bleiben Pakete im Versandsystem stecken. Wenn deine Bestellung irgendwo in den Vereinigten Staaten festsitzt, kann sie möglicherweise mit [USPS](https://faq.usps.com/s/article/USPS-Tracking-The-Basics) nachverfolgt werden. Dazu kann normalerweise die zuständige Poststelle kontaktiert werden. Wenn die Bestellung außerhalb der Vereinigten Staaten festsitzt, kontaktiere den Postdienst deines Landes für weitere Hilfe.

Reviewer: Note that this paragraph is not about normal tracking, it's about packages getting stuck somewhere along the way. Jason is right that you'd use the tracking website/service provided by the USPS even if local handling is done by German Post or DHL. But you wouldn't contact a USPS office if your package gets stuck somewhere in a DHL sorting center, I think.

It's probably not that deep. Even the current text is *fine*, as it does direct you to a national post service

> Gelegentlich bleiben Pakete im Versandsystem stecken. Wenn deine Bestellung irgendwo in den Vereinigten Staaten festsitzt, kann sie möglicherweise mit [USPS](https://faq.usps.com/s/article/USPS-Tracking-The-Basics) nachverfolgt werden. Dazu kann normalerweise die zuständige Poststelle kontaktiert werden. Wenn die Bestellung außerhalb der Vereinigten Staaten festsitzt, kontaktiere den Postdienst deines Landes für weitere Hilfe.

Author: I will leave the text as-is then if you think it's okay.

We can adjust anytime if people complain later on.

> Kontaktiere gerne [merch@ppy.sh](mailto:merch@ppy.sh), wenn du noch weitere Fragen hast. Beachte jedoch, dass nachdem wir ein Produkt versandt haben, wir genau so viel über den Lieferstatus wissen wie du.

Reviewer: ```suggestion
Kontaktiere gerne [merch@ppy.sh](mailto:merch@ppy.sh), wenn du noch weitere Fragen hast. Beachte jedoch, dass wir genau so viel über den Lieferstatus wissen wie du, nachdem wir ein Produkt versandt haben.
```

## Ich benötige Hilfe mit einem bestimmten Produkt!

### osu! keycaps {id=osu!-keycaps}

> [osu! keycaps](https://www.thekapco.com/products/osu-pippi-keycap-set) (Tastenkappen) werden von The KapCo hergestellt und vertrieben, wir empfehlen bei Problemen mit Keycaps also [mit deren Kundensupport](https://www.thekapco.com/pages/faqs) zu reden.

Reviewer: ```suggestion
[osu! keycaps](https://www.thekapco.com/products/osu-pippi-keycap-set) (Tastenkappen) werden von The KapCo hergestellt und vertrieben. Deshalb empfehlen wir bei Problemen mit Keycaps, sich an [ihren Kundensupport](https://www.thekapco.com/pages/faqs) zu wenden.
```

Sollte der Kundensupport keine zufriedenstellende Antwort liefern, kontaktiere [merch@ppy.sh](mailto:merch@ppy.sh) und wir werden sehen, was wir tun können.

### osu! deskmat {id=osu!-deskmat}

> Während viele osu!-Produkte von unserem Warenlager in den Vereinigten Staaten versandt werden, werden [osu! deskmats](https://osu.ppy.sh/store/products/1347) (Tischunterlagen) von verschiedenen Verteilerzentren im Vereinigten Königreich, in den Vereinigten Staaten und in Australien vertrieben. Das heißt, wir haben nicht über jeden Artikel, der geliefert wird, einen genauen Überblick.

Reviewer: ```suggestion
Während viele osu!-Produkte von unserem Warenlager in den Vereinigten Staaten versandt werden, werden [osu! deskmats](https://osu.ppy.sh/store/products/1347) (Tischunterlagen) von verschiedenen Verteilerzentren im Vereinigten Königreich, in den Vereinigten Staaten und in Australien versandt. Das heißt, wir haben nicht über jeden Artikel, der geliefert wird, einen genauen Überblick.
```
